redfish-core/lib/log_services.hpp

```diff
@@ -1211,24 +1438,53 @@ inline void createDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
         BMCWEB_LOG_DEBUG("Dump Created. Path: {}", objPath.str);
         createDumpTaskCallback(std::move(payload), asyncResp, objPath);
     },
-        "xyz.openbmc_project.Dump.Manager", getDumpPath(dumpType),
+        "xyz.openbmc_project.Dump.Manager", getDumpPath(createDumpType),
         "xyz.openbmc_project.Dump.Create", "CreateDump", createDumpParamVec);
 }
 
 inline void clearDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                       const std::string& dumpType)
 {
-    crow::connections::systemBus->async_method_call(
-        [asyncResp](const boost::system::error_code& ec) {
+    std::string dumpInterface;
+    if (dumpType == "Resource" || dumpType == "Hostboot" ||
+        dumpType == "Hardware" || dumpType == "SBE")
+    {
+        dumpInterface = "com.ibm.Dump.Entry." + dumpType;
+    }
+    else
+    {
+        dumpInterface = "xyz.openbmc_project.Dump.Entry." + dumpType;
+    }
+
+    std::string dumpTypeLowerCopy;
+    std::ranges::transform(dumpType, std::back_inserter(dumpTypeLowerCopy),
+                           bmcweb::asciiToLower);
+
+    std::array<std::string_view, 1> interfaces = {dumpInterface};
+    dbus::utility::getSubTreePaths(
+        "/xyz/openbmc_project/dump/" + dumpTypeLowerCopy, 0, interfaces,
+        [asyncResp,
+         dumpType](const boost::system::error_code& ec,
+                   const dbus::utility::MapperGetSubTreePathsResponse&
+                       subTreePaths) { // certPaths) {
         if (ec)
```

Comment:
I think you may want to remove the comment part.
```
-                       subTreePaths) { // certPaths) {
+                       subTreePaths) { 
```

Reply:
yes done